src/components/Tender.vue

```diff
@@ -0,0 +1,484 @@
+              <div
+              id="text"
+              style="width: 50rem"
+              @click="openDialog(contract), dialog = true"
+              > {{ contract.name }} </div>
+              </v-chip
+              ></v-item>
+            <v-item v-if="!isContract">
+              <v-text-field
+              single-line label="* Contract"
+              variant="outlined"
+              density="compact"
+              ></v-text-field>
+              <input
+              input
+              id="contract-input"
+              class="d-none"
+              type="file"
+              accept="application/pdf"
+              @change="onFileChangedContract">
+              <v-btn
+              color="primary"
+              rounded="0"
+              height="40"
+              width="150">
+              <label
+              class="file-label"
+              for="contract-input"
+              >Upload</label>
+            </v-btn
+            ></v-item>
+          </v-row>
+
+          <v-row class="mt-5 mx-8">
+            <v-item v-if="isAwardDecision">
+              <v-chip
+              size="large"
+              class="mb-6"
+              closable
+              color="blue"
+              prepend-icon="mdi-file-document-multiple-outline"
+              label @click:close="isAwardDecision = false"
+              ><div
+              id="text"
+              style="width: 50rem"
+              @click="openDialog(awardDecision), dialog = true"
+              > {{ awardDecision.name }} </div>
+              </v-chip>
+            </v-item>
+            <v-item v-if="!isAwardDecision">
+              <v-text-field
+              single-line
+              label="* Award decision"
+              variant="outlined"
+              density="compact"
+              ></v-text-field>
+              <input
+              id="award-decision-input"
+              class="d-none"
+              type="file"
+              accept="application/pdf"
+              @change="onFileChangedAwardDecision"
+              ><v-btn
+              color="primary"
+              rounded="0"
+              height="40"
+              width="150"
+              ><label
+              class="file-label" for="award-decision-input"
+              >Upload</label
+              ></v-btn>
+            </v-item>
+          </v-row>
+          <v-row class="mt-5 mx-8">
+            <v-item v-if="isRejectDecision">
+              <v-chip
+              size="large"
+              class="mb-6"
+              closable
+              color="blue"
+              prepend-icon="mdi-file-document-multiple-outline"
+              label @click:close="isRejectDecision = false"
+              ><div
+              id="text"
+              style="width: 50rem"
+              @click="openDialog(rejectDecision), dialog = true"
+              > {{ rejectDecision.name }} </div>
+              </v-chip>
+            </v-item>
+            <v-item v-if="!isRejectDecision">
```

Comment:
better to rename it isRejectDecision --> isRejectedDecision since we focus here on the final result